sqlglot/expressions.py

```diff
@@ -3093,6 +3093,10 @@ class IncludeProperty(Property):
     arg_types = {"this": True, "alias": False, "column_def": False}
 
 
+class ForceProperty(Property):
+    arg_types = {"this": True}
```

Comment:
Why do you need an argument for this? You could leave this empty (`{}`) and instead just generate the expression trivially as `"FORCE"`.